src/onConsentChange.ts

```diff
@@ -51,9 +49,7 @@ export const onConsentChange: (fn: Callback) => void = (callBack) => {
 	// if consentState is already available, invoke callback immediately
 	getConsentState()
 		.then((consentState) => {
-			if (consentState) {
-				invokeCallback(newCallback, consentState);
-			}
+			invokeCallback(newCallback, consentState);
```

Comment:
So `consentState` is always defined?

Reply:
that's what typescript thinks. i wonder if we should update the type so it can be `undefined`?

Reply:
In TS we trust.